src/components/connections/ConnectionLogoImage.tsx

```diff
@@ -23,18 +25,35 @@ const ConnectionLogoImage = ({
 }: Props) => {
     if (appLogo) {
         return (
-            <div
-                className={twMerge(
-                    cn('h-5 w-5 flex-shrink-0 overflow-hidden bg-white dark:bg-light-black', {
-                        'border-[10px] border-solid': withBorder,
-                        'border-white dark:border-light-black': withBorder,
-                        'rounded-full': roundCorners,
-                    }),
-                    className,
-                )}
-            >
-                <img className='h-full w-full  object-contain' src={appLogo} alt={alt || appName} />
-            </div>
+            <Color src={appLogo} crossOrigin='anonymous' format='hex'>
```

Comment:
this doesn't seem to apply to the "connected apps" list as it still has the grey background there at all times